examples/shader/Program.cs

```diff
@@ -13,10 +13,7 @@ protected Effect(string name)
             myName = name;
         }
 
-        public string Name
-        {
-            get { return myName; }
-        }
+        public string Name => myName;
```

Comment:
I see no reasons to stick with older version of C#, if port to .NET Core, better to stick with modern C#